rwengine/src/data/Weather.hpp

```diff
@@ -40,6 +40,14 @@ class Weather {
                       float a, float tod);
 
     std::vector<Entry> entries;
+
+    // Taken from: https://www.gtamodding.com/wiki/Time_cycle#Weather_lists
+    // TODO: This weather list applies only for GTA III
+    const uint16_t WeatherList[64] = {
```

Comment:
maybe the array could include the fact it's GTA3 specific? prefix or postfix with game name or game codename? What do you think?

Reply:
For now I would just leave the TODO there; the consensus has been (or was couple years back when the project was more active) that we should prioritize getting GTA III into playable state.

I hope that someone who implements the VC/SA part, refactors this properly.

Reply:
I would love to spend time (at some point) to getting miami working.
May I propose special mark in comment? maybe `gta3-specific`?

 Would be nice to have +- track (`grep`able) of all the places, where the engine needs to be adjusted.